cranelift/codegen/build.rs

```diff
@@ -16,6 +16,7 @@
 
 use cranelift_codegen_meta as meta;
 
+use sha2::{Digest, Sha512};
```

Comment:
md5 or sha1 should be enough for this purpose, right? That is probably faster.

Reply:
Speed is not a concern here so few files are hashed.